packages/private_key/src/lib.rs

```diff
@@ -16,17 +17,25 @@ impl PrivateKeyWASM {
             Err(err) => Err(err),
         }
     }
-}
 
-#[wasm_bindgen]
-impl PrivateKeyWASM {
+    #[wasm_bindgen(js_name = "fromBytes")]
+    pub fn from_bytes(bytes: Vec<u8>, network: NetworkWASM) -> Result<Self, JsValue> {
```

Comment:
By the way. why do we even need it here, however we now have signByBytes which does not accept a network
whats the purpose of the network here?